.github/workflows/release.yaml

```diff
@@ -70,7 +70,7 @@ jobs:
 
     - name: Ref Type
       id: ref-type
-      run: cargo run --package ref-type -- --reference ${{ github.ref }}
+      run: cargo run --package ref-type -- --reference ${{ github.ref }}  # Please provide a suggestion
```

Comment:
I am unsure what changes need to be made.  Can you make a ___\`\`\`suggestion___ or point me to some docs?

Reply:
I think you just need to do:

```
    - name: Ref Type
      id: ref-type
      run: cargo run --package ref-type -- --reference ${{ github.ref }} >> $GITHUB_OUTPUT
```

Once you change it, I can run the release workflow on a tag, to make sure it works.

Reply:
```suggestion
      run: cargo run --package ref-type -- --reference ${{ github.ref }} >> $GITHUB_OUTPUT
```